src/Nox.Types/Types/Markdown/Markdown.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Nox.Types;
+
+/// <summary>
+/// Represents a Nox <see cref="Markdown"/> type and value object.
+/// </summary>
+public sealed class Markdown : ValueObject<string, Markdown>
+{
+    public new static Markdown From(string value)
+    {
+        value = value.Trim();
+
+        var newObject = new Markdown
+        {
+            Value = value,
+        };
+
+        var validationResult = newObject.Validate();
+
+        if (!validationResult.IsValid)
+        {
+            throw new TypeValidationException(validationResult.Errors);
+        }
+
+        return newObject;
+    }
+
+    private readonly Dictionary<string, string> _htmlTag = new()
```

Comment:
Let's move this dictionary (possible tuple) to the MarkdownTag.cs

Reply:
Dictionary changed for a tuple.